rebar.config

```diff
@@ -24,7 +24,8 @@
         {dev_mode, true},
         {include_erts, false},
 
-        {overlay, [{copy, "REVISION", "REVISION"}]},
+        {overlay, [{copy, "REVISION", "REVISION"},
+                   {copy, "VERSION" , "VERSION"}]},
```

Comment:
Not clear how creates VERSION in the first place?

Reply:
VERSION is in https://github.com/aeternity/epoch/blob/master/VERSION

Reply:
😓 thanks